modular_skyrat/modules/importedvendors/code/vendors.dm

```diff
@@ -0,0 +1,182 @@
+/obj/effect/spawner/random/vending/snackvend
+	loot = list(
+		/obj/machinery/vending/imported,
+		///obj/machinery/vending/imported/panslav,
+		/obj/machinery/vending/imported/yangyu,
+		/obj/machinery/vending/imported/mothic,
+		/obj/machinery/vending/imported/tizirian,
+	)
+
+/obj/machinery/vending/imported
+	name = "NT Sustenance Supplier"
+	desc = "A vending machine serving up only the finest of human college student food."
+	icon = 'modular_skyrat/modules/importedvendors/icons/imported_vendors.dmi'
+	icon_state = "ntfood"
+	panel_type = "panel15"
+	light_mask = "ntfood-light-mask"
+	light_color = LIGHT_COLOR_LIGHT_CYAN
+	product_slogans = "Caution, contents may be selling hot!;Look at these low prices!;Hungry? Me too, wait no you didn't hear that!"
```

Comment:
```suggestion
	product_slogans = "Caution, contents may be selling hot!;Look at these low prices!;Hungry? Me too, wait were you writing that down?"
```
If they didn't want it recorded they'd simply stop it during the recording; plus, they aren't recorded, they're programmed as evidenced by haywire machines having slogans (Why would people voice hacked voicelines?).

The implication that the suggestion made it through writing and screening and THEN got voiced/programmed would be very funny still tho.

Reply:
the implication is that the machine is sentient (all vendors are if the brand intelligence event has anything to say about it)